grammars/erlang.bnf

```diff
@@ -321,6 +324,32 @@ argument_definition_list ::= '(' argument_definition? (',' argument_definition)*
 clause_guard ::= when <<guarded guard>> {pin=1}
 clause_body ::= '->' exprs {pin=1}
 
+app_expression ::= empty '{' 'application' ',' module_ref ',' app_parameters '}' {elementType=tuple_expression}
```

Comment:
This is what I want to happen. 
If I remove `empty`, it will become pin on `{` anyway (because of tuple_expression element type). If it pins on '{', parser generator adds '{' to variants list, you know.
I haven't found anything like "unpin". So I decide to pin on empty. I have tested it, all errors have logical and clear text.